test/integration/controller/core/admissioncheck_controller_test.go

```diff
@@ -26,14 +26,24 @@ import (
 
 	kueue "sigs.k8s.io/kueue/apis/kueue/v1beta1"
 	utiltesting "sigs.k8s.io/kueue/pkg/util/testing"
+	"sigs.k8s.io/kueue/test/integration/framework"
 	"sigs.k8s.io/kueue/test/util"
 )
 
 // +kubebuilder:docs-gen:collapse=Imports
 
-var _ = ginkgo.Describe("AdmissionCheck controller", func() {
+var _ = ginkgo.Describe("AdmissionCheck controller", ginkgo.Ordered, func() {
```

Comment:
```suggestion
var _ = ginkgo.Describe("AdmissionCheck controller", ginkgo.Ordered, ginkgo.ContinueOnFailure, func() {
```

Can we continue test cases even if any tests fail?

Reply:
done